gsad/src/gsad_http.c

```diff
@@ -390,7 +388,8 @@ handler_send_response (http_connection_t *connection,
                                content_disposition);
     }
 
-  add_caching_headers (response, allow_caching);
+  if (response_data->allow_caching == 0)
```

Comment:
change this access to use a getter function like cmd_response_data_is_allow_caching